lua/wire/stools/motor.lua

```diff
@@ -182,10 +185,18 @@ function TOOL:RightClick( trace )
 end
 
 function TOOL:Reload( trace )
-	if not IsValid( trace.Entity ) or trace.Entity:IsPlayer() then return false end
-	if CLIENT then return true end
-
-	return constraint.RemoveConstraints( trace.Entity, "WireMotor" )
+	if self:GetStage() > 0 then
+		self:ClearObjects()
+		self:SetStage(0)
+		if CLIENT then
+			self:ReleaseGhostEntity()
```

Comment:
Isn't the ghost entity serverside in singleplayer? Should this be shared?

Reply:
It seemed like it worked as client in both an SP and listen(self-host) MP, but it's probably for the best to move it to both realms